From 60d91a0755dfa7f45836655e56b33865b10f927b Mon Sep 17 00:00:00 2001
From: Valerii Chubar <valerii_chubar@epam.com>
Date: Thu, 4 Mar 2021 18:20:23 +0200
Subject: [PATCH] domd:u-boot: Remove reserved memory in device-tree

Signed-off-by: Valerii Chubar <valerii_chubar@epam.com>
Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
---
 arch/arm/dts/r8a7795-salvator-x-4x2g.dts | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/dts/r8a7795-salvator-x-4x2g.dts b/arch/arm/dts/r8a7795-salvator-x-4x2g.dts
index 5b4edf6129..2ce47eda18 100644
--- a/arch/arm/dts/r8a7795-salvator-x-4x2g.dts
+++ b/arch/arm/dts/r8a7795-salvator-x-4x2g.dts
@@ -31,8 +31,19 @@
 		device_type = "memory";
 		reg = <0x7 0x00000000 0x0 0x80000000>;
 	};
+
+    reserved-memory {
+        /delete-node/linux,adsp@57000000;
+        /delete-node/linux,cma@58000000;
+        /delete-node/linux,multimedia@70000000;
+    };
+
+    /delete-node/mmngr;
+    /delete-node/mmngrbuf;
 };
 
+/delete-node/&adsp;
+
 &pciec0 {
 	/* Map all possible DDR as inbound ranges */
 	dma-ranges = <0x42000000 0 0x40000000 0 0x40000000 0 0x80000000>;
-- 
2.17.1

